server/services/involvedStaffService.js

```diff
@@ -2,159 +2,180 @@ const moment = require('moment')
 const logger = require('../../log.js')
 const { ReportStatus } = require('../config/types')
 
-/**
- * @param {object} args
- * @param {any} args.incidentClient
- * @param {import('../types/uof').UserService} args.userService
- * @param {any} args.statementsClient
- * @param {any} args.db
- */
-module.exports = function createReportService({ incidentClient, statementsClient, userService, db }) {
-  const getDraftInvolvedStaff = reportId => incidentClient.getDraftInvolvedStaff(reportId)
-
-  const removeMissingDraftInvolvedStaff = async (userId, bookingId) => {
-    const { id, form = {} } = await incidentClient.getCurrentDraftReport(userId, bookingId)
-
-    const { incidentDetails = {} } = form
-    const { involvedStaff = [] } = incidentDetails
-    const updatedInvolvedStaff = involvedStaff.filter(staff => !staff.missing)
-
-    const updatedFormObject = {
-      ...form,
-      incidentDetails: { ...incidentDetails, involvedStaff: updatedInvolvedStaff },
-    }
-    await incidentClient.updateDraftReport(id, null, updatedFormObject)
-  }
-
-  const getInvolvedStaff = reportId => incidentClient.getInvolvedStaff(reportId)
+const AddStaffResult = {
+  SUCCESS: 'success',
+  SUCCESS_UNVERIFIED: 'unverified',
+  MISSING: 'missing',
+  ALREADY_EXISTS: 'already-exists',
+}
```

Comment:
add `.freeze()` ?   Not really needed, but emphasises that these are immutable values.  I should do this myself!

Reply:
I have done that before.. I'll do in a follow up